framework/components/ecl-forms/ecl-forms-selects/ecl-forms-selects.config.js

```diff
@@ -26,6 +35,7 @@ module.exports = {
       context: {
         id: 'example-selectid-2',
         is_disabled: true,
+        options: optionsList,
```

Comment:
Small tip: you can define a common context to all variants (with `context`). Check http://fractal.build/guide/core-concepts/configuration-files#configuration-inheritance